src/vbk/pop_service/pop_service_impl.cpp

```diff
@@ -71,6 +72,9 @@ PopServiceImpl::PopServiceImpl()
         LogPrintf("-------------------------------------------------------------------------------------------------\n");
         StartShutdown();
     }
+    else {
+        setConfig();
```

Comment:
does setConfig remove state stored in alt-service?

Reply:
Not at all. The state of the sqllite database with the veriblock blocks and bitcoin blocks will not be removed.